source/documentation/values/calculations.html.md.erb

```diff
@@ -161,42 +171,55 @@ CSS to unitless numbers:
   @debug calc(-infinity) < math.$min-number  // true
 <% end %>
 
-## `min()` and `max()`
+## [Calculation Functions]
+ 
+* Comparison Functions: [`min()`], [`max()`], and [`clamp()`]
+* Stepped Value Functions: [`round()`], [`mod()`], and [`rem()`].
+* Trigonometric Functions: [`sin()`], [`cos()`], [`tan()`], [`asin()`], [`acos()`], [`atan()`], and [`atan2()`].
+* Exponential Functions: [`pow()`], [`sqrt()`], [`hypot()`], [`log()`], and [`exp()`].
+* Sign-Related Functions: [`abs()`] and [`sign()`].
 
-<% impl_status dart: ">=1.11.0 <1.42.0", libsass: false, ruby: false, feature: "Special function syntax" do %>
-  LibSass, Ruby Sass, and versions of Dart Sass prior to 1.11.0 *always* parse
-  `min()` and `max()` as Sass functions. To create a plain CSS `min()` or
-  `max()` call for those implementations, you can write something like
-  `unquote("min(#{$padding}, env(safe-area-inset-left))")` instead.
-
-  Versions of Dart Sass between 1.11.0 and 1.40.0, and between 1.40.1 and 1.42.0
-  parse `min()` and `max()` functions as [special functions] if they're valid
-  plain CSS, but parse them as Sass functions if they contain Sass features
-  other than interpolation, like variables or function calls.
-
-  Dart Sass 1.41.0 parses `min()` and `max()` functions as calculations, but
-  doesn't allow unitless numbers to be combined with numbers with units. This
-  was backwards-incompatible with the global `min()` and `max()` functions, so
-  that behavior was reverted.
-
-  [special functions]: ../syntax/special-functions
-<% end %>
+[Calculation Functions]: https://www.w3.org/TR/css-values-4/#math
+[`min()`]: ../modules/math#min
+[`max()`]: ../modules/math#max
+[`clamp()`]: ../modules/math#clamp
+[`round()`]: https://developer.mozilla.org/en-US/docs/Web/CSS/round
+[`sin()`]: https://developer.mozilla.org/en-US/docs/Web/CSS/sin
+[`cos()`]: https://developer.mozilla.org/en-US/docs/Web/CSS/cos
+[`tan()`]: https://developer.mozilla.org/en-US/docs/Web/CSS/tan
+[`asin()`]: https://developer.mozilla.org/en-US/docs/Web/CSS/asin
+[`acos()`]: https://developer.mozilla.org/en-US/docs/Web/CSS/acos
+[`atan()`]: https://developer.mozilla.org/en-US/docs/Web/CSS/atan
+[`atan2()`]: https://developer.mozilla.org/en-US/docs/Web/CSS/atan2
+[`pow()`]: https://developer.mozilla.org/en-US/docs/Web/CSS/pow
+[`sqrt()`]: https://developer.mozilla.org/en-US/docs/Web/CSS/sqrt
+[`hypot()`]: https://developer.mozilla.org/en-US/docs/Web/CSS/hypot
+[`log()`]: https://developer.mozilla.org/en-US/docs/Web/CSS/log
+[`abs()`]: https://developer.mozilla.org/en-US/docs/Web/CSS/abs
+[`exp()`]: https://developer.mozilla.org/en-US/docs/Web/CSS/exp
+[`mod()`]: https://developer.mozilla.org/en-US/docs/Web/CSS/mod
+[`rem()`]: https://developer.mozilla.org/en-US/docs/Web/CSS/rem
+[`sign()`]: https://developer.mozilla.org/en-US/docs/Web/CSS/sign
+
+## [Special Cases]
```

Comment:
"Special Cases" doesn't communicate very much—it doesn't gie the reader any hint which cases are special, or what's special about them. I'd call this something like "Legacy Global Functions".